plugins/connections/ui/src/main/java/org/pentaho/di/vfs/connections/ui/dialog/ConnectionDialog.java

```diff
@@ -0,0 +1,411 @@
+/*******************************************************************************
+ *
+ * Pentaho Data Integration
+ *
+ * Copyright (C) 2002-2022 by Hitachi Vantara : http://www.pentaho.com
```

Comment:
If this is entirely new code I think we're just supposed to have the current year on it.